doc/en/how-to/capture-warnings.rst

```diff
@@ -42,8 +42,18 @@ Running pytest now produces this output:
     -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
     ======================= 1 passed, 1 warning in 0.12s =======================
 
-The ``-W`` flag can be passed to control which warnings will be displayed or even turn
-them into errors:
+Controlling warnings
+--------------------
+
+Similar to Python's `warning filter` and :option:`-W option <python:-W>` flag, pytest provides
+its own ``-W`` flag to control which warnings are ignored, displayed, or turned into
+errors. See the `warning filter_` documentation for more
```

Comment:
The underscore outside the backticks, and don't forget the first one:

```suggestion
Similar to Python's `warning filter`_ and :option:`-W option <python:-W>` flag, pytest provides
its own ``-W`` flag to control which warnings are ignored, displayed, or turned into
errors. See the `warning filter`_ documentation for more
```

https://pytest--9464.org.readthedocs.build/en/9464/how-to/capture-warnings.html#controlling-warnings